routes/bot.js

```diff
@@ -6,7 +6,7 @@ import sortRandomly from '../helpers/sortRandomly.js'
 export default async function (fastify) {
   fastify.post(
     '/bot',
-    { onRequest: [fastify.zoom.verifyRequest] },
+    { preValidation: [fastify.zoom.verifyRequest] },
```

Comment:
I wanted this to work out of the box, but I couldn't figure out how to populate `req, res` in here